### PR TITLE
Delete a Task (#58)

### LINKED ISSUE #58: Delete a Task
# Description

Create interface to delete tasks

# Suggestion

- Method: DELETE
- Also delete the solution

### PR DESCRIPTION
**Pull request by PAException**
## Description
Issue [#58](https://github.com/E-Edu/task/issues/58)

## Type of change

- [ ] Bug fix
- [x] Feature request
- [ ] Documenting code

## Checklists

- [x] I referenced the according Issue
  - [x] I created an Issue
  - [x] I put a link in the Section [Related Issue](#related-issue)
- [ ] I tested my code with multiple browsers
- [x] My commit messages are good and understandable

### CODE REVIEW

File: src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java [hunk 1]
```diff
@@ -47,6 +47,12 @@ public TaskModel createTask(
 		return this.taskRepository.save(taskModel);
 	}
 
+	public Object deleteTask(int taskId) {
+		taskRepository.deleteById(taskId);
```
Comment (steve-hb) [inline src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java]: this.

File: src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java [hunk 2]
```diff
@@ -47,6 +47,12 @@ public TaskModel createTask(
...
+		//TODO delete solution
```
Comment (steve-hb) [inline src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java]: TODO:

File: src/main/java/de/themorpheus/edu/taskservice/endpoint/TaskEndpoint.java [hunk 3]
```diff
@@ -53,4 +54,11 @@ public Object verifyTask(@PathVariable("taskId") int taskId) {
 		return this.taskController.verifyTask(taskId);
 	}
 
+	@DeleteMapping("/task/{taskId}")
+	public Object deleteTask(@PathVariable("taskId") int taskId) {
```
Comment (steve-hb) [inline src/main/java/de/themorpheus/edu/taskservice/endpoint/TaskEndpoint.java]: try @Min(0) instead of manual validation

**Review by steve-hb — CHANGES_REQUESTED** [review]

File: src/main/java/de/themorpheus/edu/taskservice/endpoint/TaskEndpoint.java (hunk 3 above)
Comment (PAException) [inline src/main/java/de/themorpheus/edu/taskservice/endpoint/TaskEndpoint.java]: delete '("taskId")'

**Pull request closed without merging**